### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Alessandranitti15**
https://Alessandranitti15.github.io/layout_product-cards-en/
https://Alessandranitti15.github.io/layout_product-cards-en/report/html_report/

### CODE REVIEW

File: src/button.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.product-card__button {
```
[the remark this answers is not in the document]
Comment (Alessandranitti15) [reply src/button.css:1]: Screenshot 2022-10-18 at 18 59 57
Hello! I tried to create a block folder but now the page is broken. To me the path looks correct

[the remark this answers is not in the document]
Comment (Alessandranitti15) [reply src/button.css:1]: Screenshot 2022-10-18 at 19 04 07
Hi, I tried to create the block folder and I think the path is correct but my page is broken :( I don't understand why

File: src/rating.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.product-card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+}
```
Comment (amitpatiljc) [inline src/rating.css:6]: how you changed naming for button and rating same way change for rating as well in `rating.css`

File: src/styles/blocks/product.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  text-align: left;
+}
+
```
Comment (amitpatiljc) [inline src/styles/blocks/product.css:9]: for this file name should be same as block name `product-card.css`
or class name should be `product`

**Review by amitpatiljc — CHANGES_REQUESTED** [review]

File: src/price.css [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  line-height: 18px;
+  color: #616070;
+}
+
+.product-card__price-number {
+  font-weight: 700px;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
```
Comment (amitpatiljc) [inline src/price.css:20]: delete this file as you have this file already in blocks folder

File: src/rating.css [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.product-card__rating-reviews {
+  font-style: bold;
+  font-weight: 400px;
+  font-size: 10px;
+  line-height: 14px;
+}
```
Comment (amitpatiljc) [inline src/rating.css:32]: delete this file as you have same file in blocks folder
and attach this file to your style.css

**Review by amitpatiljc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**